earth2studio/io/netcdf4.py

```diff
@@ -31,31 +32,25 @@ class NetCDF4Backend:
 
     Parameters
     ----------
-    file_name : str,
-        File name to provide for creating the netcdf4 store.
-    diskless : bool, optional
-        Whether to store the Dataset in memory buffer
-        Default value is False.
-    persist : bool, optional
-        Whether to save in-memory diskless buffer to disk upon close.
-        Default value is False.
+    file_name : str
+        _description_
+    nc_kwargs : dict[str, Any], optional
```

Comment:
Can we make this `backend_kwargs` to be comparable with `xarray`?

Reply:
Good idea, updated :+1: